packages/gatsby-plugin-pixel-facebook/src/pixel/index.tsx

```diff
@@ -1,60 +1,68 @@
 import React, { Fragment, useMemo } from 'react'
 import type { FC } from 'react'
-import { isServer } from '@vtex/gatsby-theme-store/src/utils/env'
 import { useLazyScript } from '@vtex/gatsby-theme-store/src/sdk/lazyScript/useLazyScript'
 import { usePixelEvent } from '@vtex/gatsby-theme-store/src/sdk/pixel/usePixelEvent'
 import { once } from '@vtex/gatsby-theme-store/src/sdk/once'
 
 import type { FBPixelProviderProps } from './types'
 import { handler } from './handler'
 
-const setupFBPixel = once((pixelId: string) => {
-  if (isServer) {
```

Comment:
Question (out of ignorance): when should I use this `isServer` check?

Reply:
It's not necessary in here since this only runs on the browser (because it's inlcuded on `gatsby-browser.js` file)